.circleci/config.yml

```diff
@@ -25,15 +25,16 @@ jobs:
       - restore_cache:
           name: Restore Yarn Package Cache
           keys:
-            - yarn-packages-{{ checksum "yarn.lock" }}
-
+            - npm-deps-{{ .Branch }}
+            - npm-deps-{{ checksum "package.json" }}
+            - npm-deps-}
```

Comment:
Ah, I was a little bit wrong with my suggestion. This exact pattern won't work as we expect.
We should have these keys for restoring cache:
```
- npm-deps-{{ .Branch }}-{{ checksum "package.json" }}
- npm-deps-{{ .Branch }}-
- npm-deps-
```

And we should use the following key for save_cache:
```
key: npm-deps-{{ .Branch }}-{{ checksum "package.json" }}
```